client/app/components/RouterCommentScreen.jsx

```diff
@@ -10,6 +11,12 @@ function select(state) {
 }
 
 class RouterCommentScreen extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+
+    _.bindAll(this, '_renderNotification');
```

Comment:
@mamute Can you check this is actually required, meaning a binding of this.

The reason I ask is that it looks like this._renderNotifcation is simply a helper that returns either null or a react component.

I suspect that the fix is that that the `::` was not needed.